lib/checks/aria/abstractrole-evaluate.js

```diff
@@ -1,6 +1,14 @@
 import { tokenList } from '../../core/utils';
 import { getRoleType } from '../../commons/aria';
 
+/**
+ * Check if an elements `role` attribute uses any abstract role values.
+ *
+ * Abstract roles are taken from the `ariaRoles` standards object from the roles `type` property.
+ *
```

Comment:
Should we put these into a namespace?

```suggestion
 *
 * @memberof checks
```